package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-icons",
-    "displayName": "vscode-icons",
+    "displayName": "VSCode Icons",
```

Comment:
Same here

Reply:
I think to use **VSCode Icons** is better. **vscode-icons** is only an id name. I can see you use **VSCode Icons** to `Preferences: File Icon Theme`:
![image](https://user-images.githubusercontent.com/27698189/31370196-d8cd1acc-adb3-11e7-9611-cb92a1336d34.png)

Reply:
BTW, other File Icon Themes often don't use `name`  for `displayName`.

![image](https://user-images.githubusercontent.com/27698189/31370449-60277ac0-adb5-11e7-8205-d789fdac3ea3.png)

Reply:
@torn4dom4n We have already discussed this as a team and we decided to go with @robertohuertasm reasoning of keeping the name "dev" related. For the history, I have already raised the naming issue some months ago.

Reply:
Yes, this was already discussed. We would have to dig the closed issues to get that old debate. 

@JimiC was in favour of changing it and I have to say that I've come to abandon the initial idea that I held to when we discussed this. So... if you want to open the debate again, I'm already open to discuss it. Even I've come to think that now it's really the time to get rid of the `preview` tag. 

@JimiC , @jens1o what is your opinion on this?

Reply:
I'm in awe 😮

Reply:
That's a good or bad awe?

Reply:
Neutral.

Reply:
For me, I want to use **VSCode Icons** because **vscode-icons** is an id name. And I favor removing `preview` tag, **VSCode Icons** is good enough to use.

Reply:
If used, `VSCode Icons` will still be just the display name. `vscode-icons` **is** the extensions name and will remain so even if the displayed name changes.

Reply:
VSCode has `displayName` and Atom is not. See https://code.visualstudio.com/docs/extensionAPI/extension-manifest#_fields

Reply:
@torn4dom4n Exactly what I'm saying.

Reply:
To not make this PR comment lengthy I suggest to keep only the link update and move the name change to an issue for further discussion.

Reply:
@JimiC Should I revert commit `Using VSCode icons name` and open an issue?

Reply:
You will have to keep only `Update link`. And yes opening a new issue is the way to go.

Reply:
I opened an issue to discuss at https://github.com/vscode-icons/vscode-icons/issues/1232